scripts/library.sh

```diff
@@ -327,21 +327,28 @@ function report_go_test() {
   return ${failed}
 }
 
-# Install Knative Serving in the current cluster.
-# Parameters: $1 - Knative Serving manifest.
-function start_knative_serving() {
-  header "Starting Knative Serving"
-  subheader "Installing Knative Serving"
-  echo "Installing Serving CRDs from $1"
-  kubectl apply --selector knative.dev/crd-install=true -f "$1"
-  echo "Installing the rest of serving components from $1"
-  kubectl apply -f "$1"
+# Install the stable release Knative/serving or the latest from source code in the current cluster.
+# Parameters: $1 - Knative Serving version number or latest.
+function start_release_knative_serving() {
```

Comment:
I suggest you keep start_knative_serving() and create something like start_knative_serving_version() that will simply call start_knative_serving with the full URL for a versioned manifest. No need to touch start_latest_knative_serving.

Reply:
No need to handle "latest". It's redundant since your doing exactly what start_latest_knative_serving() is doing.

Reply:
I will remove it.